bahmni-docker/docker-compose.yml

```diff
@@ -145,14 +150,18 @@ services:
   bahmni-web:
     image: bahmni/bahmni-web:${BAHMNI_WEB_IMAGE_TAG:?}
     profiles: ["default","openmrs"]
-    #volumes:
-      #- "${BAHMNI_APPS_PATH:?}/ui/app/:/usr/local/apache2/htdocs/bahmni"
+    volumes:
+      - "bahmniapps-data:/usr/local/apache2/htdocs/bahmni"
```

Comment:
This volume mounting needs to be host directory mount which would allow developers to link their local code repo with docker. This should not be made as named volume.

Reply:
Updated.